webpack.config.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
     filename: 'moment-range.js',
     library: 'moment-range',
     libraryTarget: 'umd',
-    path: './dist/',
+    path: __dirname + '/dist/',
```

Comment:
sounds like the version of webpack you're using may not be the same as the [one used in development](https://github.com/rotaready/moment-range/blob/master/yarn.lock#L4143-L4145)